packages/cli/src/PublicApi/v1/handlers/credentials/credentials.service.ts

```diff
@@ -30,6 +37,18 @@ export async function getSharedCredentials(
 		relations,
 	});
 }
+export async function getCredentialsAndCount(
```

Comment:
FYI I added this because my original intent was to fully flesh out the credentials api but I decided to impliment it one part at a time if this pull is accepted. So this function is not currently used anywhere but should be eventually.